apps/accounts/verify.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+import requests
+
+log = logging.getLogger('accounts.verify')
+ProfileNotFound = LookupError
```

Comment:
just a nit but if you called this something like `FxaIdentificationError` then it would apply to both profile fetching and token fetching.